### ISSUE TITLE
Recursive clone doesn't work

### ISSUE THREAD

**Issue by SevereOverfl0w** [xtdb/xtdb#1151]
```
git clone --recursive https://github.com/juxt/crux
```

Fails with

```
Submodule 'docs/crux-site' (git@github.com:juxt/crux-site.git) registered for path 'docs/crux-site'
Cloning into '/home/overfl0w/src/github.com/juxt/crux/docs/crux-site'...
ERROR: Repository not found.
fatal: Could not read from remote repository.
```

Presumably because it's private or doesn't exit.

**Comment by refset** [comment]
Yep https://github.com/juxt/crux-site is a private repo and contains the source for everything required to build https://opencrux.com except the docs themselves (which live in this repo).

**Comment by jarohen** [comment]
fixed in #1206 - it's no longer a submodule
